updating/updating-disconnected-cluster.adoc

```diff
@@ -1,23 +1,145 @@
+
+Before you perform the mirror procedure, you must prepare the host to retrieve content
+and push it to the remote location.
+
+include::modules/cli-installing-cli.adoc[leveloffset=+2]
+
+include::modules/installation-adding-mirror-registry-pull-secret.adoc[leveloffset=+1]
+
+[IMPORTANT]
+===
+This step requires that you have write access to a container image registry on the
+mirror registry and adding those credentials to a registry pull secret. This secret
+should NOT be used to install a cluster as it would give machines within the cluster
+write access to your mirror registry.
+===
+
+=== Mirror the images to local container registry
+
+Before you update a cluster on infrastructure that you provision in a restricted network, you must mirror the required container images into that environment. You may also use this procedure in unrestricted networks to ensure your clusters only use container images that have satisfied your organizational controls on external content.
+
+.Procedure
+
+. Find out if there is link:https://access.redhat.com/solutions/4583231[an update path] that exists between your current version and the desired version for the cluster to move.
+
+. Set the required environment variables:
++
+----
+$ OCP_RELEASE=<release_version> <1>
+$ LOCAL_REGISTRY='<local_registry_host_name>:<local_registry_host_port>' <2>
+$ LOCAL_REPOSITORY='<repository_name>' <3>
+$ PRODUCT_REPO='openshift-release-dev' <4>
+$ LOCAL_SECRET_JSON='<path_to_pull_secret>' <5>
+$ RELEASE_NAME='ocp-release' <6>
+$ ARCHITECTURE=<server_architecture> <7>
+----
+<1> For `<release_version>`, specify the tag that corresponds to the version of {product-title} you want
+to update the cluster, such as `4.4.0`.
+<2> For `<local_registry_host_name>`, specify the registry domain name for your mirror
+repository, and for `<local_registry_host_port>`, specify the port that it
+serves content on.
+<3> For `<repository_name>`, specify the name of the repository to create in your
+registry, such as `ocp4/openshift4`.
+<4> The repository to mirror. For a production release, you must specify
+`openshift-release-dev`.
+<5> For `<path_to_pull_secret>`, specify the absolute path to and file name of
+the pull secret for your mirror registry that you created.
+<6> For a production release, you must specify
+`ocp-release`.
+<7> For `server_architecture`, specify the architecture of the server e.g. x86_64.
+
+. Mirror the version images to the internal container registry.
+
+** Connect the removable media to a system connected to the internet.
+** Mirror the images and configuration manifests to a directory on the removable media.
++
+----
+$ oc adm release mirror "--to-dir=${REMOVABLE_MEDIA_PATH}/mirror" quay.io/openshift-release-dev/${RELEASE_NAME}:${OCP_RELEASE}-${ARCHITECTURE} --dry-run
+----
++
+----
+$ oc adm release mirror "--to-dir=${REMOVABLE_MEDIA_PATH}/mirror" quay.io/openshift-release-dev/${RELEASE_NAME}:${OCP_RELEASE}-${ARCHITECTURE}
+----
++
+[NOTE]
```

Comment:
Also need to note need for 4.5.0 oc here as below since it's required for use of `--apply-release-image-signature`

Reply:
Added the note before the installing OC CLI module